inclusive-design/wecount.inclusivedesign.ca
# Create static routes

## Description

To resolve https://github.com/inclusive-design/wecount.inclusivedesign.ca/issues/90.

This pull request generate static pages for: the home page, /inclusion-challenges, /about, /news, /news/page/{page-num}, /views, /views/page/{pageNum}, /views/{a-post}, /tag/{tag-name}

## Steps to test

1. Run `npm run generate`;
2. In the output, the "Generating pages" section should generate all routes for pages mentioned above;
3. Run `npx serve dist` to test these pages.

**Expected behavior:**

Static pages should be generated.
